packages/cli/src/commands/providerDeregister.ts

```diff
@@ -1,13 +1,15 @@
-import { Logger } from '@prosopo/common'
-import { ProviderEnvironment } from '@prosopo/types-env'
+import { KeyringPair } from '@polkadot/keyring/types'
+import { LogLevelSchema, Logger, getLogger } from '@prosopo/common'
+import { ProsopoConfig } from '@prosopo/types'
+import { ProviderEnvironment } from '@prosopo/env'
 import { Tasks } from '@prosopo/provider'
 import { validateAddress } from './validators.js'
-export default (env: ProviderEnvironment, tasks: Tasks, cmdArgs?: { logger?: Logger }) => {
-    const logger = cmdArgs?.logger || env.logger
+export default (pair: KeyringPair, config: ProsopoConfig, cmdArgs?: { logger?: Logger }) => {
+    const logger = cmdArgs?.logger || getLogger(LogLevelSchema.Values.Info, 'cli.provider_deregister')
 
     return {
         command: 'provider_deregister',
-        description: 'Deregister a Provider',
```

Comment:
why have all descriptions been changed to describe?

Reply:
because type checking was not working before! `describe` is the correct fields. The previous fields were `description` or `descrption` spelled incorrectly!

Reply:
oh right, I thought the field was description! fair enough :)